CMakeLists.txt

```diff
@@ -20,7 +20,6 @@ if (APPLE)
 	option(ENABLE_APPLE_IMAGEIO "Use native Mac OS X frameworks for loading images" ON)
 endif()
 
-# FIXME: ????
```

Comment:
This is still a FIXME: I suggest not removing the note.

Reply:
My bad - I forgot to re-add it when reverting my changes.